### PR TITLE
[OSCI] Adds new author VJoshi380

### PR DESCRIPTION
**Pull request by VJoshi380**
### Description
Placing Vedang Joshi in the authors section of the website repository. 

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/osci_2023/vedangjoshi.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Vedang Joshi
+short_name:
+twitter: 
+github: VJoshi380
+---
+
+**Vedang Joshi** I love security through obsucrity :O
```
Comment (joshuarrrr) [inline _authors/osci_2023/vedangjoshi.markdown:8]: I'm not sure if the spelling _is_ the joke, but if not:
```suggestion
**Vedang Joshi** I love security through obscurity :O
```

File: _authors/osci_2023/vedangjoshi.markdown [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+name: Vedang Joshi
+short_name:
+twitter:
```
Comment (joshuarrrr) [inline _authors/osci_2023/vedangjoshi.markdown:4]: remove these if not needed

```suggestion
```

**Review by joshuarrrr — CHANGES_REQUESTED** [review]
A couple small suggestions. Otherwise looks good.

**Comment by dtaivpp** [comment]
@VJoshi380 once you address Josh's comments we are good to go, thank you!

**Comment by VJoshi380** [comment]
Thanks for letting me know about the issues. I've removed the unused sections and fixed the typo. Let me know if any other fixes need to be made.

**Review by dtaivpp — APPROVED** [review]
Looks good!

**Pull request merged**